Update IAB_Decisions.md (#95)
ietf/wiki.ietf.org: group/iab/IAB_Decisions.md

## Normal Process

The IAB chair (or executive director on the instructions of the chair) will 

> typically follow this process to initiate a vote:

Reviewer: This process does actually not specify when the vote starts. I think there is actually a value in the "normal" process in having the discussion first and then start the vote and not do it in parallel. That means more time is need and that why we had the expedient process separately but I at least agree that having a separate process for that has not worked perfectly well and something we need to reconsider.

Reviewer 2: I think maybe we can address this by specifying what we should aim for with regards to discussion before a vote. This discussion does not need to be in a meeting or a call, I think — people can request that, but we should be able to have robust email and slack discussions to work things out.

Author: +1 to setting norms for what's discussed before the vote is called. The important part is not to specify/require any particular mode of discussion.

Reviewer 2: Suggesting text below for this

> typically follow the following process to initiate a vote:

Reviewer 2: ```suggestion
typically follow this process to initiate a vote:
```

I now notice that "follow the following" sounds silly

- The chair emails the list of voting members to start an email discussion on

>   the topic, stating whether ballots are open or secret.

Reviewer 2: Typically we don't announce the ballot being open--it just defaults to that. I wonder if it should say "optionally indicating that the ballot will be secret."?

Author: Makes sense.
```suggestion
  the topic, optionally indicating that the ballots will be secret.
```

- Discussion and votes will continue on the voting members list, and during any
  teleconference(s) that occur.

- The vote will typically stay open until a position has been recorded for every
  single voting member, even after a majority has been reached. This ensures
  every voice can be heard. Note that it is reasonable for any voting member to
  request a few days to review available information before voting.

- At the chair's option, a vote can be held open until there is an opportunity

>   to discuss it in the next meeting. Likewise, if more than two voting members

Reviewer 3: If we are meeting once a month, this is probably not fast enough for something.

Reviewer 2: It depends on the timeline of the vote — for things like appointments we can try to schedule them to align well with meetings. But faster-moving items that are unexpected generally can't be scheduled with a meeting.

  request discussion in the next meeting, the vote will remain open until that
  discussion takes place. Here, 'next meeting' is the next regularly scheduled

>   meeting greater than 48 hours after the first request is received, or in the

Reviewer 2: ```suggestion
  meeting more than 48 hours after the first request is received, or in the
```

## Normal Process

The IAB chair (or executive director on the instructions of the chair) will 
typically follow this process to initiate a vote:

> - The chair emails the list of voting members to start an email discussion on

Reviewer 4: Is this start of an email discussion only? Or is it to start recording the votes? This sentence says email discussion but the following ones mention votes.

Author: ```suggestion
- The chair emails the list of voting members to open the vote and start an email discussion on
```

Reviewer 4: WFM
Thanks!

> - The chair emails the list of voting members to open the vote and start an email discussion on
>   the topic, optionally indicating that the ballots will be secret.

Reviewer 2: ```suggestion
- The chair emails the list of voting members to start an email discussion on the
topic, define when the voting will begin, and optionally indicate that the ballots
will be secret. In general, some reasonable amount of time (such as one week)
should be allocated for discussion prior to the vote beginning, but the vote may
begin without delay if necessary. In such cases, discussion happens in parallel
with voting.

- The chair or a delegate (such as a secretariat member) will formally start a vote
by emailing the list or as part of a teleconference or meeting. For votes that start in
a teleconference or meeting that do not record all votes, the vote will also be sent
to the email list.
```

- Discussion and votes will continue on the voting members list, and during any
  teleconference(s) that occur while the vote is open.

- The vote will typically stay open until a position has been recorded for every
  single voting member, even after a majority has been reached. This ensures
  every voice can be heard. Note that it is reasonable for any voting member to
  request a few days to review available information before voting.

> - At the chair's option, a vote can be held open until there is an opportunity
>   to discuss it in the next meeting. Likewise, if more than two voting members
>   request discussion in the next meeting, the vote will remain open until that
>   discussion takes place. Here, 'next meeting' is the next regularly scheduled
>   meeting more than 48 hours after the first request is received, or in the
>   case of an irregularly scheduled meeting, 7 days after that time.

Reviewer 2: ```suggestion
```

I don't think this is needed. Two reasons:
- If this is at the start, instead of holding the vote open until a discussion can happen in a teleconference, the chair could just introduce the topic and say that the vote will happen at the meeting.
- If this is a result of discussion going on on the topic, any more than two members want to have more discussion, the vote cannot conclude successfully anyhow.

Reviewer 2: I'm suggesting a replacement paragraph at the end that explains that 3 or more voting members can decide to defer the conclusion of the vote.

> - Absent that, if the vote has enough affirmative votes that it would pass if all
>   remaining votes were dissents, and at least 7 days have elapsed since the
>   vote was announced, then the chair can decide to declare the vote closed and
>   successful. This avoids deadlock if a voting member is unable or unwilling to vote.

Reviewer 2: ```suggestion
- If a vote has not received all votes, but has enough affirmative votes that
  it would pass if all remaining votes were dissents, and at least 7 days have
  elapsed since the vote was started, then the chair can decide to declare the
  vote closed and successful. This avoids deadlock if a voting member is unable
  or unwilling to vote.
  
- If a vote has not received all votes, and does not have enough affirmative votes
that it would pass if all remaining votes were dissents, the vote remains open.
This means that 3 or more voting members can choose to defer the conclusion
of the vote to allow for more discussion.
```

Reviewer 2: How about this?
```suggestion
- If a vote has not received all votes, but has enough affirmative votes that
  it would pass if all remaining votes were dissents, and at least 7 days have
  elapsed since the vote was started, then the chair can decide to declare the
  vote closed and successful. This avoids deadlock if a voting member is unable
  or unwilling to vote.
  
- If a vote has not received all votes, but has at least 7 affirmative votes and no
  more than 2 dissenting votes, and at least 7 days have elapsed since the vote
  was started, and the vote has also been discussed as part of a teleconference
  or meeting in which the vote on the agenda at least a week in advance, then
  the chair can decide to declare the vote closed and successful.
  
- If a vote has not received all votes, and neither of the above scenarios are met,
  the vote remains open. This means that when a certain number of voting members
  do not respond, the end of the vote will be deferred. This can be used as a
  mechanism to allow for more discussion.
```

Author: That looks good, although I might make some editorial tweaks after incorporating.

## Normal Process

The IAB chair (or executive director on the instructions of the chair) will 
typically follow this process to initiate a vote:

- The chair emails the list of voting members to start an email discussion on the
topic, define when the voting will begin, and optionally indicate that the ballots
will be secret. In general, some reasonable amount of time (such as one week)
should be allocated for discussion prior to the vote beginning, but the vote may
begin without delay if necessary. In such cases, discussion happens in parallel
with voting.

- The chair or a delegate (such as a secretariat member) will formally start a vote

> by emailing the list or as part of a teleconference or meeting. For votes that start in
> a teleconference or meeting that do not record all votes, the vote will also be sent
> to the email list.

Author: ```suggestion
by emailing the list or as part of a teleconference or meeting. When a vote starts in
a teleconference or meeting but does not record all members' positions, the vote will also be sent
to the email list.
```

Reviewer 2: Looks good!

  every voice can be heard. Note that it is reasonable for any voting member to
  request a few days to review available information before voting.

- If a vote has not received all votes, but has enough affirmative votes that
  it would pass if all remaining votes were dissents, and at least 7 days have
  elapsed since the vote was started, then the chair can decide to declare the
  vote closed and successful. This avoids deadlock if a voting member is unable
  or unwilling to vote.
  
- If a vote has not received all votes, but has at least 7 affirmative votes and no
  more than 2 dissenting votes, and at least 7 days have elapsed since the vote
  was started, and the vote has also been discussed as part of a teleconference
  or meeting in which the vote on the agenda at least a week in advance, then
  the chair can decide to declare the vote closed and successful.
  
- If a vote has not received all votes, and neither of the above scenarios are met,
  the vote remains open. This means that when a certain number of voting members
  do not respond, the end of the vote will be deferred. This can be used as a

>   mechanism to allow for more discussion.

Reviewer 2: Looks good!

## Normal Process

The IAB chair (or executive director on the instructions of the chair) will 
typically follow this process to initiate a vote:

> - The chair emails the list of voting members to start an email discussion on the

Reviewer 5: s/email discussion/discussion

Reason: I do not think we should limit discussion to email.

> topic, define when the voting will begin, and optionally indicate that the ballots

Reviewer 5: s/ballots/votes

Reason: May be stylistic but I think "ballot" like "the IAB decides to do X" and what we may optionally want to be secret is the specific votes (e.g., Jason voted YES).

will be secret. In general, some reasonable amount of time (such as one week)
should be allocated for discussion prior to the vote beginning, but the vote may
begin without delay if necessary. In such cases, discussion happens in parallel
with voting.

- The chair or a delegate (such as a secretariat member) will formally start a vote
by emailing the list or as part of a teleconference or meeting. When a vote starts in
a teleconference or meeting but does not record all members' positions, the vote will also be sent

> to the email list.

Reviewer 5: I have to say I am not a fan of email-based votes where we all reply all to cast a vote. I think it is fine to alert IAB members in some manner (Slack, teleconf, email) that they need to vote, I just would like us to evolve beyond email reply alls.

Author: I think this is a separable issue, but I agree FWIW.

- Discussion and votes will continue on the voting members list, and during any
  teleconference(s) that occur while the vote is open.

- The vote will typically stay open until a position has been recorded for every
  single voting member, even after a majority has been reached. This ensures
  every voice can be heard. Note that it is reasonable for any voting member to
  request a few days to review available information before voting.

> 

Reviewer 5: Why wouldn't we have an deadline date for a vote? Like 7 or 10 days or whatever. Otherwise, could one intransigent member prevent closure of the vote by not voting? ISTM we should have a "No Vote Received" option for this case.
